src/liblsquic/lsquic_frame_reader.c

```diff
@@ -698,11 +700,12 @@ code_str_to_reason (const char *code_str, int code_len)
     };
 
     long code;
-    char code_buf[ code_len + 1 ];
+    char * code_buf = malloc(code_len + 1 );
```

Comment:
I understand this was done because Windows doesn't support variables in array definitions (sigh).  But whether it works or not needs to be checked (at least an assert(code_buf) right after or something).  But generally a good solution.